DependencyInjection/CmfSeoExtension.php

```diff
@@ -197,21 +204,27 @@ private function loadAlternateLocaleProvider($config, ContainerBuilder $containe
             ? $this->defaultAlternateLocaleProviderId
             : $config['provider_id'];
 
+        if (!$alternateLocaleProvider) {
+            return;
+        }
 
-        if ($alternateLocaleProvider) {
-            $alternateLocaleProviderDefinition = $container->findDefinition($alternateLocaleProvider);
+        $alternateLocaleProviderDefinition = $container->findDefinition($alternateLocaleProvider);
+        if ($this->contentListenerEnabled) {
             $container
                 ->findDefinition('cmf_seo.event_listener.seo_content')
                 ->addMethodCall(
                     'setAlternateLocaleProvider',
                     array($alternateLocaleProviderDefinition)
                 );
+        }
+        if ($this->phpcrProviderEnabled) {
```

Comment:
decided to be explicit here. this is more robust to discover bugs when we mix up service names or the like.